frontend/src/pages/customer/ModalAddCustomer.js

```diff
@@ -0,0 +1,106 @@
+import React, {useState} from 'react';
+import Modal from "react-bootstrap/Modal";
+import Form from "react-bootstrap/Form";
+import Button from "react-bootstrap/Button";
+import validateCustomer from "../../validation/CustomerValidationRules";
+import ErrorMessage from "../../messages/errorMessage";
+
+function ModalAddCustomer(props) {
+
+    const [customerDto, setCustomer] = useState({
+        name: '',
+        adminEmail: ''
+    });
+    const [validError, setError] = useState([]);
+    const [errorMessage, setErrors] = useState('');
+
```

Comment:
what is the purpose of these 2 error state variables? Can we join them or if can not, make them more clear and obvious?